### PR TITLE
Push nginx containers on Travis builds

### PR DESCRIPTION
**Pull request by alexwlchan**
### What is this PR trying to achieve?

Bring our nginx images closer to our app images – rather than being pushed manually, they’re pushed automatically by Travis.

### Who is this change for?

Developers who want better deployments.

### CODE REVIEW

**Comment by alexwlchan** [comment]
Okay, I think this is ready for review. Next: embed the commit hash in the image tag.

(This is part of #483.)

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,53 @@
...
+lint-lambdas: docker-build-flake8
 	docker run -v $$(pwd)/lambdas:/data python3.6_ci:latest
+
+
+
+.PHONY: help
+
+## Display this help text
+help: # Some kind of magic from https://gist.github.com/rcmachado/af3db315e31383502660
+	$(info Available targets)
```
Comment (kenoir) [inline Makefile]: The fact these "\" don't line up 😿

Comment (alexwlchan) [inline Makefile]: Ah, tab-delimited Makefiles 😈

File: .travis/install.sh [hunk 2]
```diff
@@ -10,3 +10,9 @@ if [[ "$BUILD_TYPE" == "sbt" ]]
 then
   ./.travis/install_sbt.sh
 fi
+
+if [[ "${TASK:-foo}" == "docker-build-nginx" ]]
```
Comment (alicefuzier) [inline .travis/install.sh]: foo?

Comment (kenoir) [inline .travis/install.sh]: I think this is a default value that is just **not** "docker-build-nginx"

Comment (kenoir) [inline .travis/install.sh]: For clarity could be  "not-docker-build-nginx"

File: .travis/run.sh [hunk 3]
```diff
@@ -25,5 +25,10 @@ fi
 # Eventually, we'll be able to get rid of this check and call make directly
 # from .travis.yml.
 if ! [[ -z "${TASK+x}" ]]; then
+
+  if [[ "$TRAVIS_EVENT_TYPE" == "push" ]]; then
+    TASK="${TASK/build/deploy}"
```
Comment (kenoir) [inline .travis/run.sh]: Oooo what does this do?

**Comment by alexwlchan** [comment]
(Review markups were just comments, so I cancelled the build.)

**Pull request merged**